scripts/deploy.ts

```diff
@@ -67,16 +67,18 @@ async function deployWarpRoute() {
 
   let multiProviderConfig: ChainMap<any, IChainConnection> = {};
   for (const chain of targetChains) {
-    if (chainConfig && chainConfig[chain]) {
+    if (chainConfigs && chainConfigs[chain]) {
```

Comment:
Consider adding a third branch for when `chain` is neither in `chainConfigs` nor `chainConnectionConfigs`?